### ISSUE TITLE
Content search cannot find Chinese characters

### ISSUE THREAD

**Issue by ligq** [adirh3/Fluent-Search#690]
Content search cannot find Chinese characters
My windows is Chinese version win11 22H2 22621.755

**Comment by u-Sir** [comment]
可以试下多输入几个字😉

**Comment by u-Sir** [comment]
试了下，中文要至少3个字才能搜到

**Comment by ligq** [comment]
> 试了下，中文要至少3个字才能搜到

You are right. And I found for English we also have to input at least 3 letters to start the search.
But for the Chinese, it's a bug. A Chinese character is equivalent to an English word, not a letter. The search should start after we input one Chinese character.

**Comment by u-Sir** [comment]
> > 试了下，中文要至少3个字才能搜到
> 
> You are right. And I found for English we also have to input at least 3 letters to start the search. But for the Chinese, it's a bug. A Chinese character is equivalent to an English word, not a letter. The search should start after we input one Chinese character.

developer said "it's not only Chinese it's all languages"

**Comment by adirh3** [comment]
I might add option to use Pinyin for content search, but it would double the index size. Would you be OK with that? (Double since it will account for both English a pinyin and Chinese)

**Comment by ligq** [comment]
> I might add option to use Pinyin for content search, but it would double the index size. Would you be OK with that? (Double since it will account for both English a pinyin and Chinese)

Thank you for your quick response. Pinyin could be helpful, but it's not a perfect solution, since one pinyin generally corresponds to several Chinese characters.

**Comment by adirh3** [comment]
> > I might add option to use Pinyin for content search, but it would double the index size. Would you be OK with that? (Double since it will account for both English a pinyin and Chinese)
> 
> Thank you for your quick response. Pinyin could be helpful, but it's not a perfect solution, since one pinyin generally corresponds to several Chinese characters.

Yes I am aware of that thanks to @u-Sir :)
The thing is, searching with less than 3 letters would be extremely slow, probably like more than 6 seconds, and it will also be CPU intensive.
I might enable that for Chinese specifically in the next nightly update and see how it goes.

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 1.1.1.2. Changelog - https://github.com/adirh3/Fluent-Search/releases/tag/1.1.1.2
